docs/standard/frameworks.md

```diff
@@ -1,56 +1,144 @@
+.NET Framework | 4.6.2 | .NETFramework,Version=4.6.2 | net462 | 1.5 | N/A
+
+## Supported frameworks
+
+A framework is typically referenced by a short target framework moniker or *TFM*. In .NET Standard, this is also generalized to *TxM* to allow a single reference to multiple frameworks. The NuGet clients support the following frameworks. Equivalents are shown within brackets (`[]`).
+
+| Name | Abbreviation | TFMs/TxMs |
+| --- | --- | --- |
+| .NET Standard | netstandard | netstandard1.0 |
+| | | netstandard1.1 |
+| | | netstandard1.2 |
+| | | netstandard1.3 |
+| | | netstandard1.4 |
+| | | netstandard1.5 |
+| | | netstandard1.6 |
+| .NET Core App | netcoreapp | netcoreapp1.0 |
+| | | netcoreapp1.1 |
+| .NET Framework | net | net11 |
+| | | net20 |
+| | | net35 |
+| | | net40 |
+| | | net403 |
+| | | net45 |
+| | | net451 |
+| | | net452 |
+| | | net46 |
+| | | net461 |
+| | | net462 |
+| .NET Core | netcore | netcore [netcore45] |
+| | | netcore45 [win, win8] |
+| | | netcore451 [win81] |
+| | | netcore50 |
+| .NET MicroFramework | netmf | netmf |
+| Windows | win | win [win8, netcore45] |
+| | | win8 [netcore45, win] |
+| | | win81 [netcore451] |
+| | | win10 (not supported by Windows 10 platform) |
+| Silverlight | sl | sl4 |
+| | | sl5 |
+| Windows Phone | wp | wp [wp7] |
+| | | wp7 |
+| | | wp75 |
+| | | wp8 |
+| | | wp81 |
+| | | wpa81 |
+| Universal Windows Platform | uap | uap [uap10.0] |
+| | | uap10.0 |
+
+## Deprecated frameworks
+
+The following frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
+
+Deprecated framework | Replacement
+--- | ---
+aspnet50 | netcoreapp
+aspnetcore50 | 
+dnxcore50 | 
+dnx | 
+dnx45 | 
+dnx451 | 
+dnx452 | 
+dotnet | netstandard
+dotnet50 | 
+dotnet51 | 
+dotnet52 | 
+dotnet53 | 
+dotnet54 | 
+dotnet55 | 
+dotnet56 | 
+winrt | win
+
+## Precedence
+
+A number of frameworks are related to and compatible with one another but not necessarily equivalent:
+
+Framework | Can use
+--- | ---
+uap (Universal Windows Platform) | win81
+| | wpa81
+| | netcore50
+win (Windows Store) | winrt
+| | winrt45
+
+## .NET Standard
+
+The [.NET Standard](https://github.com/dotnet/standard) simplifies references between binary-compatible frameworks, allowing a single target framework to reference a combination of others. For background, see the [.NET Standard Library](library.md).
+
+The [NuGet Tools Get Nearest Framework Tool](https://aka.ms/s2m3th) simulates the NuGet logic used for the selection of one framework from many available framework assets in a package based on the project's framework. To use the tool, enter one project framework and one or more package frameworks. Click the **Submit** button. The tool indicates if the package frameworks you list are compatible with the project framework you provide.
```

Comment:
Yes, great catch! I'll fix it. Let's just go with `http://nugettoolsdev.azurewebsites.net/`, since the latest framework is loaded :sparkles: auto-magically :sparkles: